msresist/logistic_regression.py

```diff
@@ -19,7 +19,8 @@ def plotClusterCoefficients(ax, lr, hue=None, title=False):
         hue = "Sample"
     else:
         coefs_["Cluster"] = np.arange(coefs_.shape[0]) + 1
-    sns.barplot(ax=ax, x="Cluster", y="LR Coefficient", hue=hue, data=coefs_, color='darkblue', **{"linewidth": 0.5}, **{"edgecolor": "black"})
+    p = sns.barplot(ax=ax, x="Cluster", y="LR Coefficient", hue=hue, data=coefs_, color='darkblue', **{"linewidth": 0.5}, **{"edgecolor": "black"})
+    p.set_xticklabels(coefs_["Cluster"], size=6)
```

Comment:
You’re passing the wrong number of tick labels.

Reply:
Oops, missed the value error from the traceback. Should be fixed now.